packages/terra-table/src/subcomponents/SectionHeader.module.scss

```diff
@@ -0,0 +1,98 @@
+@import '~terra-mixins';
+
+// Section Header
+// ========================================================
+:local {
+  .section-header {
+    background-color: var(--terra-table-section-header-background-color, #e8e9ea);
+    border-top: var(--terra-table-section-header-border-top, 1px solid #dedfe0);
+    overflow: hidden;
+    padding: 0;
+    width: 100%;
+    z-index: 1;
+
+    + div {
+      border-top-color: var(--terra-table-section-header-next-border-top-color, #b2b5b6);
```

Comment:
--terra-table-section-header-next-border-top-color

technically this is the border bottom right?